addons/survey/static/src/views/components/survey_survey_action_helper/survey_survey_action_helper.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <t t-name="survey.SurveySurveyActionHelper">
+        <div class="o_view_nocontent">
+            <div class="o_nocontent_help">
+                <p>
+                    No Survey Found
+                </p><p>
+                    Pick a sample or <a action="action_load_sample_custom" class="o_survey_load_sample"
+                        t-on-click.stop.prevent="() => this.onStartFromScratchClick()">Start from scratch</a>.
+                </p>
+
+                <div class="row mb-4 w-100 px-5 justify-content-center">
+                    <t t-set="templateNames" t-value="Object.keys(surveyTypeTemplateData)"/>
```

Comment:
Actually, in this commit, I would not use `type` in the name of methods / variables, overall. It seems more confusing than helping (`_prepare_live_session_type_template_values` -> `_prepare_live_session_template_values` (same for others), `surveyTypeTemplateData` -> `surveyTemplateData`)

Sorry if it came from one of my suggestions :disguised_face: 

Thanks!